src/libs/ck-libs/NDMeshStreamer/NDMeshStreamer.ci

```diff
@@ -33,14 +33,24 @@ module NDMeshStreamer {
   };
 
   template<class dtype, class ClientType, class RouterType, int (*EntryMethod)(char *, void *) = defaultMeshStreamerDeliver<dtype,ClientType> >
-    group [migratable] ArrayMeshStreamer :
-    MeshStreamer<dtype, RouterType> {
+  group [migratable] GroupMeshStreamer : MeshStreamer<dtype, RouterType> {
+    entry GroupMeshStreamer(
+        int numDimensions, int dimensionSizes[numDimensions],
```

Comment:
Perhaps this and the corresponding block for `ArrayMeshStreamer` can be unified as a #define or something since they seem to be identical. There may also be a way to bake these default params into the parent class, but perhaps that gets wonky since this is a CI.

Reply:
I should mention that the default parameters are determined in the charmxi code (`xi-Entry.C`, IIRC) so having them in the CI file and `NDMeshStreamer.h` is redundant. These are removed in this PR.